edx_proctoring/admin.py

```diff
@@ -334,6 +342,254 @@ def save_model(self, request, review, form, change):
+        """
+        List of values to allow admin to select
+        """
+        lookups = (())
+        unique_course_ids = ProctoredExamStudentAttempt.objects.values_list(
+            'proctored_exam__course_id',
+            flat=True
+        ).distinct()
+
+        for course_id in unique_course_ids:
+            lookups += ((course_id, prettify_course_id(course_id)),)
+
+        return lookups
+
+    def queryset(self, request, queryset):
+        """
+        Return the filtered queryset
+        """
+        if self.value():
+            return queryset.filter(proctored_exam__course_id=self.value())
+        else:
+            return queryset
+
+
+class ProctoredExamAttemptForm(forms.ModelForm):
+    """
+    Admin Form to display for reading/updating a Proctored Exam Attempt
+    """
+
+    class Meta(object):  # pylint: disable=missing-docstring
+        model = ProctoredExamStudentAttempt
+        fields = '__all__'
+
+    STATUS_CHOICES = [
+        (ProctoredExamStudentAttemptStatus.eligible, 'Eligible'),
+        (ProctoredExamStudentAttemptStatus.created, 'Create'),
+        (ProctoredExamStudentAttemptStatus.download_software_clicked, 'Download Software Clicked'),
+        (ProctoredExamStudentAttemptStatus.ready_to_start, 'Ready To Start'),
+        (ProctoredExamStudentAttemptStatus.started, 'Started'),
+        (ProctoredExamStudentAttemptStatus.ready_to_submit, 'Ready To Submit'),
+        (ProctoredExamStudentAttemptStatus.declined, 'Declined'),
+        (ProctoredExamStudentAttemptStatus.timed_out, 'Time Out'),
+        (ProctoredExamStudentAttemptStatus.submitted, 'Submitted'),
+        (ProctoredExamStudentAttemptStatus.second_review_required, 'Second Review Required'),
+        (ProctoredExamStudentAttemptStatus.verified, 'Verified'),
+        (ProctoredExamStudentAttemptStatus.rejected, 'Rejected'),
+        (ProctoredExamStudentAttemptStatus.not_reviewed, 'Not Reviewed'),
+        (ProctoredExamStudentAttemptStatus.error, 'Error'),
+    ]
+
+    status = forms.ChoiceField(choices=STATUS_CHOICES)
+
+
+class ProctoredExamStudentAttemptAdmin(admin.ModelAdmin):
+    """
+    Admin panel for Proctored Exam Attempts
+    """
+
+    readonly_fields = [
+        'user',
+        'proctored_exam',
+        'started_at',
+        'completed_at',
+        'last_poll_timestamp',
+        'last_poll_ipaddr',
+        'attempt_code',
+        'external_id',
+        'allowed_time_limit_mins',
+        'taking_as_proctored',
+        'is_sample_attempt',
+        'student_name',
+        'review_policy_id',
+        'is_status_acknowledged'
+    ]
+
+    list_display = [
+        'username',
+        'exam_name',
+        'course_id',
+        'attempt_code',
+        'status',
+        'modified_timestamp'
+    ]
+
+    search_fields = [
+        'user__username',
+        'attempt_code'
+    ]
+
+    list_filter = [
+        ExamAttemptFilterByStatus,
+        ExamAttemptFilterIsProctored,
```

Comment:
We can simply use the file name, and Django admin will detect that it's a boolean field and render the default boolean filter . Just add 'taking_as_proctored' and 'is_sample_attempt'. We can then remove these filter classes altogether.